package.json

```diff
@@ -27,8 +27,13 @@
       "ts"
     ],
     "transform": {
-      "^.+\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+      "^.+\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js",
```

Comment:
> "^.+\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js",

I guess it will be nice to add a comment re this wird line

Reply:
Can't comment .json files.

Reply:
I would note that this line is no less weird than many others in that file and especially in webpack.config.js.